users.html

```diff
@@ -203,6 +203,9 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // Caltech
   L.marker([34.13765314049858, -118.12527436670602], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>California Institute of Technology</strong><br>Pasadena, California, USA<ul><li><a href='https://wennberglab.caltech.edu/' target='_new'>Wennberg Lab</li></ul>");
 
+  // Carlton University
+  L.marker([45.38713780095911, -75.69583922979173], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Carlton University</strong><br>Ottowa, Ontario, Canada<ul><li><a href='https://carleton.ca/cee/cu-people/amir-hakami/' target='_new'>Department of Civil and Environmental Engineering</li></ul>");
```

Comment:
One more update, this should be "Carleton University" (it's in the link).  Sorry I missed this before!